symengine/polys/cancel.h

```diff
@@ -0,0 +1,47 @@
+#ifndef SYMENGINE_CANCEL_H
+#define SYMENGINE_CANCEL_H
+
+#include <symengine/basic.h>
+#include <symengine/polys/basic_conversions.h>
+
+#ifdef HAVE_SYMENGINE_FLINT
+using SymEngine::UIntPolyFlint;
```

Comment:
I don't think the `UIntPolyFlint` is being used at all here, so I think you can remove the `HAVE_SYMENGINE_FLINT` ifdef completely?

Reply:
Done @certik